docs/pages/getting-started.mdx

```diff
@@ -56,109 +61,43 @@ cd teleport
-$ sudo mv teleport.yaml /etc
+$ sudo teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o
+Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
 ## Step 1c: Configure Domain Name and obtain TLS certificates using Let's Encrypt
 
 Teleport requires a secure public endpoint for the Teleport UI and for end users to connect to.
-A domain name and TLS certificates are also required. We'll use Let's Encrypt to obtain a free TLS certificate.
-
-DNS Setup:<br />
-For this setup, we'll simply use an `A` or `CNAME` record pointing to the IP/FQDN of the machine with Teleport installed.
-
-TLS Setup:<br />
-If you already have TLS certificates available you can use those. If using a new domain we recommend using `certbot`, which is free and
-simple to set up. Follow [certbot instructions](https://certbot.eff.org/) for how to obtain a certificate for your distro.
-
-<Admonition type="tip" title="Using Certbot to obtain Wildcard Certs">
-Let's Encrypt provides free wildcard certificates. Below is an example command to
-use [certbot](https://certbot.eff.org/) with DNS challenge.
-Replace `foo@example.com` with your email address.
-Replace `teleport.example.com` with the domain name you want to use to access Teleport.
-
-  ```sh
-  certbot certonly \
-    --manual \
-    --preferred-challenges=dns \
-    --agree-tos \
-    --manual-public-ip-logging-ok \
-    --email foo@example.com \
-    -d "teleport.example.com, *.teleport.example.com"
-  ```
-</Admonition>
-
-**Update `teleport.yaml`**<br />
-Once you've obtained certificates from Let's Encrypt, the below commands will update Teleport's
-config file to use your newly configured domain and TLS certificates.
+To get started use an `A` or `CNAME` record pointing to the IP/FQDN of the machine with Teleport installed.
 
-Replace `teleport.example.com` with the domain name you configured above.
+<Admonition type="tip" title="Tip">
+You can use `dig` to make sure that DNS record is propagated:
 
 ```bash
-# Replace `teleport.example.com` with your domain name.
-export TELEPORT_PUBLIC_DNS_NAME="teleport.example.com"
-cat >> /etc/teleport.yaml <<EOF
-    public_addr: $TELEPORT_PUBLIC_DNS_NAME:3080
-    https_keypairs:
-    - key_file: /etc/letsencrypt/live/$TELEPORT_PUBLIC_DNS_NAME/privkey.pem
-      cert_file: /etc/letsencrypt/live/$TELEPORT_PUBLIC_DNS_NAME/fullchain.pem
-EOF
+dig @8.8.8.8 tele.example.com
 ```
+</Admonition>
 
-Once you've updated the config file, assuming you have configured with [systemd](https://github.com/gravitational/teleport/tree/master/examples/systemd), you should restart Teleport to pick up the changes:
-
-`sudo systemctl restart teleport`
-
-Otherwise you can start Teleport directly:
-
-`sudo teleport start`
+Start Teleport:
 
-You can access Teleport's web UI on port 3080.
-Replace `teleport.example.com` with your domain: `https://teleport.example.com:3080/`
+```bash
+sudo teleport start
```

Comment:
I would suggest we recommend starting with -d as it provides better debug info when getting started. 
```suggestion
sudo teleport start -d
```

Reply:
Yeah, if we're not emitting enough information to be useful on first startup at `INFO` level then we should fix that in Teleport.